src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -122,6 +122,29 @@ public FeedbackSessionResultsBundle(FeedbackSessionAttributes feedbackSession,
         // roster.*Table is populated using the CourseRoster data directly
         this.rosterTeamNameMembersTable = getTeamNameToEmailsTableFromRoster(roster);
         this.rosterSectionTeamNameTable = getSectionToTeamNamesFromRoster(roster);
+        if (!areQuestionNumbersConsistent()) {
+            log.severe(feedbackSession.getIdentificationString() + " has invalid question numbers");
+        }
+    }
+    
+
+    //TODO can be removed once we are sure that question numbers will be consistent
+    private boolean areQuestionNumbersConsistent() {
+        Set<Integer> questionNumbersInSession = new HashSet<>();
+        for (FeedbackQuestionAttributes question : this.questions.values()) {
+            if (questionNumbersInSession.contains(question.questionNumber)) {
+                return false;
+            }
+            questionNumbersInSession.add(question.questionNumber);
```

Comment:
This returns false if the set already contains the element. We could use that behaviour in the if condition above I guess.